roles/atomic_scan/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# vim: set ft=ansible:
+#
+#   This role verifies whether container image doesn't exist CVEs.
+#
+- name: Fail if image is undefined
+  fail:
+    msg: "Image is undefined"
+  when: image is undefined
+
+- name: Set scanner image (RHEL)
+  set_fact:
+    scanner_image: "registry.access.redhat.com/rhel7/openscap"
+
+- name: Install openscap
+  command: atomic --debug install {{ scanner_image }}
+  register: result
+
+- name: Check installation of openscap
+  fail:
+    msg="Atomic install was unsuccessful"
```

Comment:
nit, can you change the = to : to make it consistent with the fail message above?

`msg: "Atomic install was unsuccessful" `